src/tokens2.jl

```diff
@@ -66,7 +66,16 @@ end
          ii[2].address == 1?                       2 :
          ii[2].address == 2?                       3 : 1
 
+"""
+    compare(m::SAContainer, s::IntSemiToken, t::IntSemiToken)
 
+### Args:
+* `m`: SortedDict or SortedMultiDict.
+* `s`: Semitoken
+* `t`: Semitoken
+
+This function determines the relative positions of the data items indexed by (m,s) and (m,t) in the sorted order.
```

Comment:
I'm not familiar with this at all, but I must say I don't understand from this description what the function does. Maybe detail a bit more? Or give an example?